src/app/features/help/routes.tsx

```diff
@@ -0,0 +1,9 @@
+import HelpPage from "./components/pages/Help/HelpPage"
+
+export default {
+  "/hulp": {
+    Page: HelpPage,
+    title: "Hulp",
+    icon: "Handicap"
```

Comment:
Handicap? :-) Wouldn't a questionmark be better?

Reply:
QuestionMark is not available from ASC. Have made a Jira task to discuss this.